addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -170,5 +182,22 @@ weSnippetEditor.Class.include({
             tab: this.tabs.THEME,
         });
     },
+    /**
+     * Called when search input value changed > adapts the snippets grid.
+     */
+    _onSearchFilterInputChange: function () {
+        var self = this;
+        self._searchFilterValue = self.$('.o_snipet_search_filter_input').val().toLowerCase();
+        // show all snippets
+        self.$('.o_panel').show();
+        self.$('.oe_snippet').filter(function(){
+            var $snippet = $(this);
+            $snippet.toggle(
+                $snippet.attr('name').toLowerCase().indexOf(self._searchFilterValue) > -1
+            );
```

Comment:
Never use the jQuery `toggle` / `hide` / `show` class. Instead toggle the `d-none` class on the element.